Bottom: c7d277b2ee00baa57451024961b637cfba5df7fc
Top:    f1f46a9fecbb2ddf69ede5f91adac965d372d981
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 00:07:37 +0100

djopt.c: Fixup warnings due to assigned but unused variables


---

diff --git a/src/djopt.c b/src/djopt.c
index 0b00d8e..d25d621 100644
--- a/src/djopt.c
+++ b/src/djopt.c
@@ -948,7 +948,6 @@ static int
 split_line (line_s * l, corner_s * c)
 {
   int i;
-  LayerType *layer;
   LineType *pcbline;
   line_s *ls;
 
@@ -965,7 +964,6 @@ split_line (line_s * l, corner_s * c)
     }
 
   check (c, l);
-  layer = PCB->Data->Layer + l->layer;
   pcbline = create_pcb_line (l->layer,
 			     c->x, c->y, l->e->x, l->e->y,
 			     l->line->Thickness, l->line->Clearance,
@@ -2558,7 +2556,6 @@ pinsnap ()
   PinType *pin;
   int again = 1;
 
-  corner_s *prev_c;
   int close = 0;
   corner_s *c2;
 
@@ -2575,7 +2572,6 @@ pinsnap ()
 	  if (!(c->pin || c->via || c->pad))
 	    continue;
 
-	  prev_c = c;
 	  pin = 0;
 
 	  dprintf ("\ncorner %s\n", corner_name (c));
